docs/modeling/example-fitting-constraints.rst

```diff
@@ -47,38 +63,91 @@ Fitters support constrained fitting.
 
 Bounded fitting is supported through the ``bounds`` arguments to models or by
 setting `~astropy.modeling.Parameter.min` and `~astropy.modeling.Parameter.max`
 attributes on a parameter.  Bounds for the
 `~astropy.modeling.fitting.LevMarLSQFitter` are always exactly satisfied--if
 the value of the parameter is outside the fitting interval, it will be reset to
-the value at the bounds. The `~astropy.modeling.fitting.SLSQPLSQFitter` handles
-bounds internally.
+the value at the bounds. The `~astropy.modeling.fitting.SLSQPLSQFitter` optimiztion
+algorithm handles bounds internally.
 
-- Different fitters support different types of constraints::
+.. _tied:
 
-    >>> fitting.LinearLSQFitter.supported_constraints
-    ['fixed']
-    >>> fitting.LevMarLSQFitter.supported_constraints
-    ['fixed', 'tied', 'bounds']
-    >>> fitting.SLSQPLSQFitter.supported_constraints
-    ['bounds', 'eqcons', 'ineqcons', 'fixed', 'tied']
+Tied Constraints
+----------------
+
+The `~astropy.modeling.Parameter.tied` constraint is often useful with :ref:`Compound models <compound-models-intro>`.
+In this example we will read a spectrum from a file called ``spec.txt``
+and fit Gaussians to the lines simultaneously while linking the flux of the OIII_1 and OIII_2 lines.
+
+.. plot::
+    :include-source:
+       
+    >>> import numpy as np
+    >>> from astropy.io import ascii
+    >>> from astropy.utils.data import get_pkg_data_filename
+    >>> from astropy.modeling import models, fitting
+    >>> fname = get_pkg_data_filename('data/spec.txt', package='astropy.modeling.tests')
+    >>> spec = ascii.read(fname)
+    >>> wave = spec['lambda']
+    >>> flux = spec['flux']
+    
+    Use the rest wavelengths of known lines as initial values for the fit.
+
+
+    >>> Hbeta = 4862.721
+    >>> OIII_1 = 4958.911
+    >>> OIII_2 = 5008.239
+
+    Create Gaussian1D models for each of the Hbeta and OIII lines.
+
+    >>> h_beta = models.Gaussian1D(amplitude=34, mean=Hbeta, stddev=5)
+    >>> o3_2 = models.Gaussian1D(amplitude=170, mean=OIII_2, stddev=5)
+    >>> o3_1 = models.Gaussian1D(amplitude=57, mean=OIII_1, stddev=5)
+
+
+    Tie the ratio of the intensity of the two OIII lines.
+
+    >>> def tie_ampl(model):
+    ...     return model.amplitude_2 / 3.1
+    >>> o3_1.amplitude.tied = tie_ampl
+
+
+    Also tie the wavelength of the Hbeta line to the OIII wavelength.
+
+    >>> def tie_wave(model):
+    ...     return model.mean_0 * OIII_1 / Hbeta
+    >>> o3_1.mean.tied = tie_wave
+
+    Create a Polynomial model to fit the continuum.
+
+    >>> mean_flux = flux.mean()
+    >>> cont = np.where(flux > mean_flux, mean_flux, flux)
+    >>> linfitter = fitting.LinearLSQFitter()
+    >>> poly_cont = linfitter(models.Polynomial1D(1), wave, cont)
+    
+    Create a compound model for the three lines and the continuum.
+
+    >>> hbeta_combo = h_beta + o3_1 + o3_2 + poly_cont
+    
+    Fit all lines simultaneously.
+
+    >>> fitter = fitting.LevMarLSQFitter()
```

Comment:
@nden , try remove all the `>>>` from the code *inside* the plot directive. The example that you said was successful did not have the `>>>`.